scripts/run-spark.py

```diff
@@ -84,7 +98,7 @@ def _get_app_args(args):
         for step in STEPS:
             args_dict[step] = True
     args.total_data_size = int(args.total_gb * 10 ** 9)
-    args.num_mappers = int(np.ceil(args.total_data_size / args.input_part_size))
+    args.num_mappers = int(np.ceil(args.total_data_size / (args.input_part_size * 10 ** 9)))
```

Comment:
Maybe rename to `args.input_part_gb` and derive `input_part_size` as we did with total data size above.